cmd/cireport/main.go

```diff
@@ -20,71 +20,89 @@ var (
 	username    string
 )
 
+var valid_jobs = []string{
+	"release-openshift-ocp-installer-e2e-openstack-4.4",
+	"release-openshift-ocp-installer-e2e-openstack-serial-4.4",
+	"release-openshift-ocp-installer-e2e-openstack-4.3",
+	"release-openshift-ocp-installer-e2e-openstack-serial-4.3",
+	"release-openshift-ocp-installer-e2e-openstack-4.2",
+	"release-openshift-ocp-installer-e2e-openstack-serial-4.2",
+	"release-openshift-origin-installer-e2e-openstack-4.2",
+	"release-openshift-origin-installer-e2e-openstack-serial-4.2",
+}
+
 func main() {
-	ids, err := sequence.Int(jobIDs)
-	if err != nil {
-		panic(err)
+	client := gsheets.NewClient()
+
+	var jobs []string
+	if fullJobName == "" {
+		jobs = valid_jobs
+	} else {
+		jobs = append(jobs, fullJobName)
 	}
 
-	for _, i := range ids {
-		j := job.Job{
-			FullName: fullJobName,
-			ID:       strconv.Itoa(i),
+	for _, jobName := range jobs {
+		sheet := gsheets.Sheet{
+			JobName: jobName,
+			Client:  &client,
 		}
 
-		startedAt, err := j.StartTime()
-		if err != nil {
-			panic(err)
+		var realJobIDs string
+		if jobIDs == "" {
+			lowerBound := sheet.GetLatestId() + 1
+			upperBound := prow.GetLatestId(jobName)
+			if lowerBound < upperBound {
+				realJobIDs = fmt.Sprintf("%d-%d", lowerBound, upperBound)
+			} else if lowerBound == upperBound {
+				realJobIDs = fmt.Sprintf("%d", upperBound)
```

Comment:
We should be [covered][1] already

[1]: https://github.com/pierreprinetti/go-sequence/blob/350b74325d6470e42f48e089d1a2ead45bab71a3/sequence_test.go#L93-L99